cfgov/unprocessed/css/molecules/global-banner.less

```diff
@@ -38,7 +38,9 @@
     }
 
     .o-expandable_target {
-        .btn__secondary();
+        // Taken from .a-btn__secondary.
```

Comment:
This one doesn't have any limitation, it should definitely be moved to using the default class instead of duplicating the styles.